tests/test_dataset_common.py

```diff
@@ -207,8 +210,10 @@ def test_dataset_has_valid_etag(self, dataset_name):
 
     def test_builder_class(self, dataset_name):
         builder_cls = self.dataset_tester.load_builder_class(dataset_name)
-        builder = builder_cls()
-        self.assertTrue(isinstance(builder, DatasetBuilder))
+        name = builder_cls.BUILDER_CONFIGS[0].name if builder_cls.BUILDER_CONFIGS else None
+        with tempfile.TemporaryDirectory() as tmp_cache_dir:
```

Comment:
I added used a temp dir here @patrickvonplaten as the builder tries to load its `.info` from the cache in `__init__`